redcap_bridge/test_redcap/test_server_interface.py

```diff
@@ -0,0 +1,108 @@
+import json
+
+import pandas as pd
+import pytest
+
+import redcap
+from redcap_bridge.server_interface import upload_datadict, download_records
+from redcap_bridge.test_redcap.test_utils import (test_directory,
+                                                  initialize_test_dir)
+from redcap_bridge.utils import map_header_csv_to_json
+
+SERVER_CONFIG_YAML = (
+        test_directory / 'testfiles' / 'TestProject' / 'project.json').resolve()
+
+
+@pytest.fixture
```

Comment:
I'm not sure if I understand the fixture thing

Reply:
Fixtures are a pytest concept to create specific states for tests. I am not an expert myself, but I guess it is similar to the unittest setUp etc...